pkgs/development/python-modules/textual-dev/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, aiohttp
+, buildPythonPackage
+, click
+, fetchFromGitHub
+, msgpack
+, poetry-core
+, pythonOlder
+, textual
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "textual-dev";
+  version = "1.2.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "Textualize";
+    repo = pname;
+    rev = "fb5aa869784895066dbd0ec850d92ac9205cde73";
+    hash = "sha256-WO2uYw9rUvEMlhir3T8UA8VXGN/BU7Arn4Nxim1j5k0=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    aiohttp
+    click
+    msgpack
+    textual
+    typing-extensions
+  ];
+
+  pythonImportsCheck = [
+    "textual_dev"
+  ];
+
+  meta = with lib; {
+    description = " Development tools for Textual ";
```

Comment:
```suggestion
    description = "Development tools for Textual";
```

Reply:
Ahh missed this sorry, fixed

Reply:
fixed this in original commit when rebasing